pkgs/applications/video/mpv/scripts/mpvacious.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchzip, curl, xclip }:
+
+stdenv.mkDerivation rec {
+  pname = "mpvacious";
+  version = "v0.12";
+  src = fetchzip {
+    url = "https://github.com/Ajatt-Tools/mpvacious/archive/${version}.tar.gz";
+    sha256 = "1xz4qh2ibfv03m3pfdasim9byvlm78wigx1linmih19vgg99vky2";
```

Comment:
```suggestion
  version = "0.12";

  src = fetchzip {
    url = "https://github.com/Ajatt-Tools/mpvacious/archive/v${version}.tar.gz";
    sha256 = "1xz4qh2ibfv03m3pfdasim9byvlm78wigx1linmih19vgg99vky2";
```

Reply:
I originally had it this way but wasn't confident. :smile: